traffic_ops/build/build_rpm.sh

```diff
@@ -76,25 +76,15 @@ function initBuildArea() {
                 { echo "Could not build atstccfg binary"; exit 1; }
```

Comment:
Need to remove the part that builds `atstccfg` from the TO build script.

Reply:
hm. Weird. I know I tested that after my rebase.